src/styles/sidebar/components/focused-mode-header.scss

```diff
@@ -1,11 +1,26 @@
+// A dark grey button used for the primary action
+// in a form
 .focused-mode-header {
-  margin-bottom: 10px;
-  button {
-    width: 100%;
-    height: 24px;
-    margin-right: 10px;
-    &:focus {
-      outline: none;
+  display: flex;
+  align-items: center;
+
+  &__btn {
```

Comment:
I've noticed that the ordering of rules for this selector is sloppy. I'll fix as part of fixup round.